.github/workflows/create-sdk-update-pr.yml

```diff
@@ -0,0 +1,64 @@
+name: Create SDK update PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      sdk_name:
+        description: 'The SDK for which the version should be updated'
+        required: true
+        type: choice
+        options:
+          - android
+          - ios
+      version_number:
+        description: 'The version number to which the SDK should be updated'
+        required: true
+        type: string
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ github.token }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Git User
+        run: |
+          git config --global user.name "Update Bot"
+          git config --global user.email "update-bot@bitmovin.com"
+
+      - name: Set Update Branch name
```

Comment:
NIT: job names are not consistent with upper and lower case characters.

Reply:
https://github.com/bitmovin/bitmovin-player-react-native/pull/87/commits/ac102b1321b860928978cc50d7f4c3eeec03e766